docs/APIClient.md

```diff
@@ -0,0 +1,208 @@
+# API Client
+
+The API client is a set of Python libraries that can be used to interact with the REST API of Timesketch from notebooks or scripts. It takes
+care of setting up authentication, sending the API calls to the server, error handling, and presentation.
+
+This is not meant to be a full fledged documentation of the API client, but rather a starting point for people on how to use the API client in
```

Comment:
Maybe something like:

Reply:
maybe something like?????

Reply:
Hm interesting that the proposal is gone

"This documentation will give an overview for the most common use cases. Some methods implemented in the API client might not be covered here".

Idea of that change is to start the sentence positive.

Reply:
SGTM